docs/conf/australia/2021/news/cancel-announcement.rst

```diff
@@ -0,0 +1,41 @@
+:template: {{year}}/generic.html
+
+.. post:: August 25, 2021
+   :tags: {{shortcode}}-{{year}}, announcement, conference
+
+Cancelling WTD Australia and India 2021 conference
+===================================================
+
+G'day folks, we have decided to cancel our WTD Australia and India conference in 2021.
+While this has been an extremely difficult decision, we recognise the immense mental fatigue that lockdowns have caused in Australia and India this year.
+
+While the community engagement and support have been fantastic across our remote meetup events and last year's conference, we have all felt stressed and burnt-out at various times during the last couple of years.
+Keeping in mind the long(er)-term health of the community, we have decided to do something else in place of the conference.
+
+Here's what we will do instead.
+
+Mega meetup
```

Comment:
Maybe "mega" isn't the word that we want to use externally? I don't have a better suggestion....

Reply:
Super?